src/pages/vagrant/docs/[[...page]].tsx

```diff
@@ -0,0 +1,55 @@
+import { ReactElement } from 'react'
+import vagrantData from 'data/vagrant.json'
+import { Product } from 'types/products'
+import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import DocsView from 'views/docs-view'
+import Button from '@hashicorp/react-button'
+import fetchContentApiFileString from 'lib/fetch-content-api-file-string'
+import { GetStaticPropsContext, GetStaticPropsResult } from 'next'
+
+const basePath = 'docs'
+const baseName = 'Docs'
+const product = vagrantData as Product
+const additionalComponents = { Button }
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const VagrantDocsPage = ({ mdxSource }): ReactElement => {
+  return <DocsView {...mdxSource} additionalComponents={additionalComponents} />
+}
+
+const { getStaticPaths } = getStaticGenerationFunctions({
+  product,
+  basePath,
+  baseName,
+})
+
+// TODO: We need to pass VMWARE_UTILITY_VERSION as { scope } in order to render
```

Comment:
Would love input on this workaround solution. My intent here isn't to have a final solution, but rather to get things working for now (🛹 skateboard!), then keep on adding products, and then coming back to this to make it more robust later on. Very open to suggestions on how to achieve that direction, or to alternate directions 🙇‍♂️

Reply:
Ah nice, that seems very viable! Will fetch from https://releases.hashicorp.com/vagrant-vmware-utility/index.json & use that data to parse the latest version 💯

Reply:
✅ Have refactored to fetch from the releases API, and applied Bryce's suggestion from Slack to use an async `getScope` getter as the new arg in `getStaticGenerationFunctions`, rather than a `scope` object as I had previously. Feels much cleaner to me 😍 Let me know what you think! 🙇‍♂️